Documentation (#7)
marmelab/solid-admin: packages/admin/docs/Show.md

# `<Show>`

The `<Show>` component is responsible for displaying a record.

It leverages the dataProvider `getOne` method.

> It's an headless components that provide its features through the `RecordContext` and leave the UI responsibility to its children.

Reviewer: ```suggestion
It's an headless components that provides its features through the `RecordContext` and leaves the UI responsibility to its children.
```

## Props

- `resource`: Optional. The name of the targeted resource. It will be inferred from the `ResourceContext`.
- `id`: Optional. The identifier of the record to edit. Defaults to the one provided in the route parameters.
- `meta`: Optional. Metadata to pass to the dataProvider methods.

> - `queryOptions`: Optional. Solid-Query options for the `getOne` request.

Reviewer: same

## Example

```jsx
import { Show, NumberField, SimpleForm, TextField, BooleanField } from '@solid-admin/admin';

export const PostShow = () => {
	return (
		<div class="flex flex-col p-4 gap-4">
			<Show>
				<SimpleForm>
					<TextField source="title" />
					<TextField source="teaser" />
					<TextField source="body" />
					<NumberField source="views" />
					<BooleanField source="commentable" />
				</SimpleForm>

> 			</Show>

Reviewer: ```suggestion
			<Show>
				<Label source="title">
					<TextField source="title" />
				</Label>
				<Label source="teaser">
					<TextField source="teaser" />
				</Label>
				<Label source="commentable">
					<BooleanField source="commentable" />
				</Label>
			</Show>
```

(and need to update the imports)

# `<Show>`

The `<Show>` component is responsible for displaying a record.

It leverages the dataProvider `getOne` method.

It's an headless components that provides its features through the `RecordContext` and leaves the UI responsibility to its children.

## Props

- `resource`: Optional. The name of the targeted resource. It will be inferred from the `ResourceContext`.
- `id`: Optional. The identifier of the record to edit. Defaults to the one provided in the route parameters.
- `meta`: Optional. Metadata to pass to the dataProvider methods.
- `queryOptions`: Optional. [Solid-Query options](https://tanstack.com/query/latest/docs/react/reference/useQuery) for the `getOne` request.

## Example

```jsx

> import { Show, NumberField, SimpleForm, TextField, BooleanField } from '@solid-admin/admin';

Reviewer: ```suggestion
import { Show, Label, TextField, BooleanField } from '@solid-admin/admin';
```
